doc/asciidoc/algorithms.adoc

```diff
@@ -1,15 +1,15 @@
 = Algorithms
 
-Graph algorithms are used to compute metrics for graphs, nodes or relationships.
+Graph algorithms are used to compute metrics for graphs, nodes, or relationships.
 
-They can provide insights on relevant entities (centralities, ranking) in the graph or inherent structures like communities (community-detection, graph-partitioning, clustering).
+They can provide insights on relevant entities in the graph (centralities, ranking), or inherent structures like communities (community-detection, graph-partitioning, clustering).
 
-Many of them are iterative approaches that frequently traverse the graph for the computation using random walks, breadth-first- or depth-first searches or pattern matching.
+Many graph algorithms are iterative approaches that frequently traverse the graph for the computation using random walks, breadth-first or depth-first searches, or pattern matching.
 
 Due to the exponential growth of possible paths with increasing distance, many of the approaches also have high algorithmic complexity.
 
-Fortunately optimized algorithms exist that utilize certain structures of the graph, memoize already explored parts, and parallelize operations.
-Whenever possible we've applied these optimizations.
+Fortunately, optimized algorithms exist that utilize certain structures of the graph, memorize already explored parts, and parallelize operations.
```

Comment:
This is actually supposed to say `memoize` as in they cache the explored parts of the graph so they don't need to revisit them.

Reply:
cool - I'll revert 😃